src/include/btmem.h

```diff
@@ -851,6 +851,7 @@ struct __wt_page {
     uint64_t evict_pass_gen;       /* Eviction pass generation */
     uint64_t evict_queue_attempts; /* Number of times eviction tries to queue a page for eviction
                                       but fails */
+    uint16_t evict_page_attempts;  /* Number of times eviction tries to evict a page */
```

Comment:
Sorry, I think we should also make other stats as 16bit. Otherwise, we are not saving any space here. I think we can make evict_queue_attempts as 16 bit as well. If we can only make two variables 16bit. We should move them to the end of the struct.